widget.subscribe.php

```diff
@@ -62,22 +62,20 @@ public function __construct() {
 		);
 
 		// Only register widget scripts, styles, and ajax when widget is active
-		if ( is_active_widget( false, false, $this->id_base, true ) ) {
```

Comment:
This line was causing the shortcode to not work entirely because this function returns false for shotcode.

Reply:
What does disabling this do?  Is there an edge case that you're now not preventing from being handled? What happens if no forms are on the page?

Reply:
it shouldn't break anything -- either widget is valid and this script only runs when the subscribe widget is created one way or another.

Reply:
jk, i found [this](https://developer.wordpress.org/reference/functions/shortcode_exists/) function. adding back the if statement.